test_util/aws.py

```diff
@@ -67,6 +68,7 @@ class CfStack():
     def __init__(self, stack_name, boto_wrapper):
         self.boto_wrapper = boto_wrapper
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(stack_name)
+        self.wait_for_stack_creation()
```

Comment:
this shouldn't implicitly wait for stack creation, we may be connecting to an already created stack